benzina/src/__private/mod.rs

```diff
@@ -0,0 +1,7 @@
+#![doc(hidden)]
```

Comment:
This works, but it's also something not many people expect to see. I'd have `#[doc(hidden)]` be on top on `mod __private;` in `lib.rs`